css/components/fields/label-items.css

```diff
@@ -0,0 +1,15 @@
+/**
+ * @file
+ * This file is used to style fields with class 'label-items', providing a global style for meta fields
+ */
+
+.label-items .field__label,
+.label-items .field__items,
+.label-items .field__item {
+  display: inline-block;
+}
+.label-items .field__label,
+.label-items .field__item {
+  margin-right: 0.6rem;
```

Comment:
Writing direction related settings should have a comment. And also a rulset for RTL languages. See: https://www.drupal.org/docs/develop/standards/css/css-formatting-guidelines#rtl

I don't know if it is needed in our special usecase (this theme is built for an Englis installation profile currently.

Reply:
Good point. Shall we raise this as a separate issue as I'm guessing this has a broad impact?